src/docs/product/data-management-settings/event-grouping/stack-trace-rules.mdx

```diff
@@ -264,3 +291,50 @@ family:native stack.package:**/proprietarymodule.so  max-frames=2
 family:native stack.function:KnownBadFunction1  max-frames=5
 family:native stack.function:KnownBadFunction2  max-frames=5
 ```
+
+### Sentinel & Prefix Frames
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+In [the Grouping Breakdown tab](src/docs/product/sentry-basics/guides/grouping-and-fingerprints/grouping-breakdown), you can adjust the level of detail by which
```

Comment:
How is this breakdown tab useful?

* can be used to determine whether stacktraces within group are largely the same
* can be used to determine whether grouping by one frame instead of all makes any difference